src/components/template-modal/screens/template-picker/index.js

```diff
@@ -22,31 +22,33 @@ export default ( { onInsertTemplate, templates } ) => {
 			<div className="newspack-newsletters-modal__content">
 				<div className="newspack-newsletters-modal__patterns">
 					<div className="block-editor-patterns">
-						{ ( templates || [] ).map( ( { title, content }, index ) => (
-							<div
-								key={ index }
-								className={
-									selectedTemplate === index
-										? 'selected block-editor-patterns__item'
-										: 'block-editor-patterns__item'
-								}
-								onClick={ () => setSelectedTemplate( index ) }
-								onKeyDown={ event => {
-									if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
-										event.preventDefault();
-										setSelectedTemplate( index );
+						{ ( templates || [] ).map( ( { title, content }, index ) =>
+							0 === index ? null : (
```

Comment:
Nitpick: similar to the other comment:
```suggestion
							"" === content ? null : (
```

Reply:
https://github.com/Automattic/newspack-newsletters/pull/130/commits/d03352e6bd1d3d71469e524c6fcccf45d0bd406d